docs/eventing/samples/kafka/channel/README.md

```diff
@@ -215,58 +252,59 @@ To use SASL authentication, you will need the following information:
 !!! note
     It is recommended to also enable TLS as described in the previous section.
 
-1. Edit your config-kafka ConfigMap:
-   ```bash
-   kubectl -n knative-eventing edit configmap config-kafka
-   ```
-2. Set the SASL.Enable field to `true`, for example:
-   ```
-   ...
-   data:
-     sarama: |
-       config: |
-         Net:
-           SASL:
-             Enable: true
-   ...
-   ```
-3. Create a secret with the username, password, and SASL mechanism, for example:
-   ```bash
-   kubectl create secret --namespace <namespace> generic <kafka-auth-secret> \
-       --from-literal=password="SecretPassword" \
-       --from-literal=saslType="PLAIN" \
-       --from-literal=username="my-sasl-user"
-   ```
+1. Edit the `config-kafka` ConfigMap:
+
+    ```bash
+    kubectl -n knative-eventing edit configmap config-kafka
+    ```
+
+1. Set the SASL.Enable field to `true`:
+
+    ```yaml
+    ...
+    data:
+      sarama: |
+        config: |
+          Net:
+            SASL:
+              Enable: true
+    ...
+    ```
+
+1. Create a secret that uses the username, password, and SASL mechanism:
+
+    ```bash
+    kubectl create secret --namespace <namespace> generic <kafka-auth-secret> \
+        --from-literal=password="SecretPassword" \
+        --from-literal=saslType="PLAIN" \
+        --from-literal=username="my-sasl-user"
+    ```
 
 ### All authentication methods
 
-1. If you have created a secret for your desired authentication method by
-   using the previous steps, reference the secret and the namespace of the
-   secret in the `config-kafka` ConfigMap:
-   ```
-   ...
-   data:
-     eventing-kafka: |
-       kafka:
+1. If you have created a secret for your desired authentication method by using the previous steps, reference the secret and the namespace of the secret in the `config-kafka` ConfigMap:
+
+    ```yaml
+    ...
+    data:
+       eventing-kafka: |
+        kafka:
          authSecretName: <kafka-auth-secret>
          authSecretNamespace: <namespace>
-   ...
-   ```
+    ...
+    ```
 
-!!! note
-    The default secret name and namespace are `kafka-cluster` and
-    `knative-eventing` respectively. If you reference a secret in a different
-    namespace, be sure you configure your roles and bindings so that the
-    knative-eventing pods can access it.
+    !!! note
+        The default secret name and namespace are `kafka-cluster` and `knative-eventing` respectively. If you reference a secret in a different namespace, be sure you configure your roles and bindings so that the `knative-eventing` pods can access it.
 
 ## Channel configuration
 
-The `config-kafka` ConfigMap allows for a variety of channel options such as:
+The `config-kafka` ConfigMap allows for a variety of Channel options such as:
 
 - CPU and Memory requests and limits for the dispatcher (and receiver for
-  the distributed channel type) deployments created by the controller
```

Comment:
A lot of parentheses in this section. Not sure if they are all necessary.

Reply:
Probably not, I'd say none of them are, but per my comment above https://github.com/knative/docs/pull/4028#issuecomment-881791577 I'm not trying to edit this to perfection in this PR - the primary reason for it is to fix the three issues linked, so for things that need additional rewriting like this or the kubectl commands as you mention in the other comment below, I'd prefer if you could raise another issue for us to clean up that stuff later so we're not blocking this PR on it. That OK?